hydra-node/test/Hydra/Chain/Direct/Contract/Mutation.hs

```diff
@@ -860,3 +860,22 @@ replaceContesters contesters = \case
       , Head.contesters = contesters
       }
   otherState -> otherState
+
+removePTFromMintedValue :: TxOut CtxUTxO -> Tx -> Value
+removePTFromMintedValue output tx =
```

Comment:
I don't understand why this one was moved here. But it's also not so important where this function lives, I guess.

Reply:
I think it's been moved when I copy/pasted all Abort mutations then realised they were not all relevant